app/src/components/OrderBookSnapshot.tsx

```diff
@@ -94,10 +97,26 @@ class OrderBookSnapshot extends Component<WithStyles, State> {
         }
     }
 
+    componentWillUnmount() {
+        window.removeEventListener('resize', debounce(this.handleResize, 100));
```

Comment:
Using a REPL, I have found that two calls to `debounce` with the same arguments do not match with `==` or `===`, therefore I am not sure if the event listener will really be removed. We could avoid this by simply storing the debounced function as an attribute on the OrderBookSnapshot object